### PR TITLE
https://issues.redhat.com/browse/ACM-4572

### PR DESCRIPTION
**Pull request by oafischer**
Also for 2.6 and 2.7.

### CODE REVIEW

File: clusters/cluster_lifecycle/create_proxy_env.adoc [hunk 1]
```diff
@@ -38,5 +38,16 @@ Replace `BMC-address-range/CIDR` with the BMC address and the number of addresse
 +
 After you add the previous values, the settings are applied to your clusters. 
 
-. Provision the cluster by completing the procedure for creating a cluster. See xref:../cluster_lifecycle/create_intro.adoc#creating-a-cluster[Creating a cluster] to select your provider.
+. Provision the cluster by completing the procedure for creating a cluster. See _Creating a cluster_ to select your provider.
+
+*Note:* After deploying your cluster, any new changes you make to `install-config` YAML do not apply. See _Managing security policies_ to learn more about making changes after deployment by using policies.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_proxy_env.adoc]: I did a search and find for `install-config.yaml` and there were no instances in the the governance book. From my perspective, making more changes is Day 2 operations, I would send the user to [MultiClusterHub advanced configuration](https://github.com/stolostron/rhacm-docs/blob/2.8_stage/install/adv_config_install.adoc)

File: clusters/cluster_lifecycle/create_proxy_env.adoc [hunk 2]
```diff
@@ -38,5 +38,16 @@ Replace `BMC-address-range/CIDR` with the BMC address and the number of addresse
...
+
+[#resources-creating-cluster-proxy]
+== Additional resources
+
+- See xref:../cluster_lifecycle/create_intro.adoc#creating-a-cluster[Creating a cluster] to select your provider.
+
+- See link:../governance/create_policy.adoc#managing-security-policies[Managing security policies] to learn how to make configuration changes after deploying your cluster.
+
+- Return to <<creating-a-cluster-proxy,Creating a cluster in a proxy environment>>.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_proxy_env.adoc]: I like how you are sending the user back to the top of the page. I would also tell the user to reference the [Cluster lifecycle introduction](https://github.com/stolostron/rhacm-docs/blob/2.8_stage/clusters/cluster_lifecycle/cluster_lifecycle_intro.adoc) for more topics.

File: clusters/cluster_lifecycle/create_proxy_env.adoc (hunk 1 above)
Comment (oafischer) [reply clusters/cluster_lifecycle/create_proxy_env.adoc]: Does that link work for you @FDewaleyne?

File: clusters/cluster_lifecycle/create_proxy_env.adoc [hunk 3]
```diff
@@ -38,5 +38,17 @@ Replace `BMC-address-range/CIDR` with the BMC address and the number of addresse
 +
 After you add the previous values, the settings are applied to your clusters. 
 
-. Provision the cluster by completing the procedure for creating a cluster. See xref:../cluster_lifecycle/create_intro.adoc#creating-a-cluster[Creating a cluster] to select your provider.
+. Provision the cluster by completing the procedure for creating a cluster. See _Creating a cluster_ to select your provider.
 
+*Note:* After deploying your cluster, any new changes you make to `install-config` YAML do not apply. See _MultiClusterHub advanced configuration_ to learn more about making changes after deployment by using policies.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_proxy_env.adoc]: After rereading the issue, we should maybe point the user to the [Supported policies file](https://github.com/stolostron/rhacm-docs/blob/2.8_stage/governance/supported_policies.adoc), where we describe that user can use pod policies for cluster workload configuration. Alternatively, we can point them directly to the [Pod policy doc](https://github.com/stolostron/rhacm-docs/blob/2.8_stage/governance/pod_policy.adoc) 

```suggestion
*Note:* After deploying your cluster, any new changes you make to `install-config` YAML do not apply. Update the cluster workload configuration after deployment by using policies. See _Pod policy_ for more information.
```

Comment (oafischer) [reply clusters/cluster_lifecycle/create_proxy_env.adoc]: Thoughts @FDewaleyne?

Comment (FDewaleyne) [reply clusters/cluster_lifecycle/create_proxy_env.adoc]: it looks good, maybe a little rephrasing to "To update the configuration post deployment, you will need to use policies. See _Pod policy_ for more information"

Comment (oafischer) [reply clusters/cluster_lifecycle/create_proxy_env.adoc]: Updated, sounds like we are good on the pod policy link.

File: clusters/cluster_lifecycle/create_proxy_env.adoc [hunk 4]
```diff
@@ -38,5 +38,17 @@ Replace `BMC-address-range/CIDR` with the BMC address and the number of addresse
 +
 After you add the previous values, the settings are applied to your clusters. 
 
-. Provision the cluster by completing the procedure for creating a cluster. See xref:../cluster_lifecycle/create_intro.adoc#creating-a-cluster[Creating a cluster] to select your provider.
+. Provision the cluster by completing the procedure for creating a cluster. See _Creating a cluster_ to select your provider.
 
+*Note:* You can only use `install-config` YAML when deploying your cluster. After deploying your cluster, any new changes you make to `install-config` YAML do not apply. To update the configuration after deployment, you must policies. See _Pod policy_ for more information.
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/create_proxy_env.adoc]:
```suggestion
*Note:* After deploying your cluster, any new changes you make to `install-config` YAML do not apply. To update the configuration after deployment, you must use policies. See _Pod policy_ for more information.
```

**Review by dockerymick — APPROVED** [review]
Left a suggestion; this looks food to me!

**Pull request merged**